app-service-template/go.mod

```
@@ -4,11 +4,11 @@ module app-new-service
 
 go 1.18
 
-replace github.com/edgexfoundry/app-functions-sdk-go/v2 => ../
+replace github.com/edgexfoundry/app-functions-sdk-go/v3 => ../
 
 require (
-	github.com/edgexfoundry/app-functions-sdk-go/v2 v2.3.0
-	github.com/edgexfoundry/go-mod-core-contracts/v2 v2.3.0
+	github.com/edgexfoundry/app-functions-sdk-go/v3 v2.3.0
```

Is replaced above so we can use the targeted release version

```suggestion
	github.com/edgexfoundry/app-functions-sdk-go/v3 v3.0.0
```